Userland/Utilities/headless-browser.cpp

```diff
@@ -209,6 +209,7 @@ static ErrorOr<String> run_one_test(HeadlessWebContentView& view, StringView inp
     } else if (mode == TestMode::Text) {
         view.on_load_finish = [&](auto const&) {
             result = view.dump_text().release_value_but_fixme_should_propagate_errors();
+            view.debug_request("collect-garbage");
```

Comment:
Seems like this adds ~20 seconds to the Linux NoFuzz CI.

Instead of adding the cost of a full GC to every single test, how about we add a `gc()` helper function to `Tests/LibWeb/Text/input/include.js`

```js
function gc() {
    // FIXME: Replace me with a window.internals.gc() helper API or similar.
	for (let i = 0; i < 100000; ++i)
        "whee";
}
```

This wastes less time on CI, and also makes it much more obvious that the test is indeed about GC effects. :)